context/app/markdown/docs/metadata.md

```diff
@@ -1,39 +1,51 @@
-|RNAseq (bulk)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/bulkrnaseq)|[Stephanie Nevins](mailto:snevins@stanford.edu)|
-|sciATAseq|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/scatacseq)|[Dana L Jackson](mailto:danaj77@uw.edu)|
-|sciRNAseq|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/scrnaseq)|[Dana L Jackson](mailto:danaj77@uw.edu)|
-|scRNAseq / 10Xgenomics|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/scrnaseq)|[Maigan Brusko](mailto:maigan@ufl.edu)|
-|SeqFISH|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/seqfish)|[Nico Pierson](mailto:nicogpt@caltech.edu)|
-|Shotgun LC-MS (targeted, lipidomics)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/lcms)|[Kevin Contrepois](mailto:kcontrep@stanford.edu)|
-|SNAREseq (ATAC)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/scatacseq)|[Blue Lake](mailto:b1lake@eng.ucsd.edu)|
-|SNAREseq (RNA)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/scatacseq)|[Blue Lake](mailto:b1lake@eng.ucsd.edu)|
-|snATACseq|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/scatacseq)|[Blue Lake](mailto:b1lake@eng.ucsd.edu)|
-|snRNAseq|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/scrnaseq)|[Stephanie Nevins](mailto:snevins@stanford.edu)|
-|TMT LC-MS (proteomics)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/lcms)|[Lihua Jiang](mailto:lihuaj@stanford.edu)|
-|Untargeted LC-MS (eg: LC-MS/MS-proteomics)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/lcms)|[Jeffrey Spraggins](mailto:jeff.spraggins@Vanderbilt.Edu)|
-|Untargeted LC-MS (metabolomics)|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/lcms)|[Kevin Contrepois](mailto:kcontrep@stanford.edu)|
-|WGS|[Document](GitHubMDFileLink)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/wgs)|[Aaron Horning](mailto:ahorning@stanford.edu)|
+**Assay**
+
+Assay|Document|Specification|Contact
+|--|--|--|--
+|AF (Autofluorescence)|[Document](https://github.com/hubmapconsortium/portal-ui/blob/master/context/app/markdown/docs/assay/af.md)|[Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/af)|[Jeffrey Spraggins](mailto:jeff.spraggins@Vanderbilt.Edu)
```

Comment:
No - The original had line-end pipes, and now they're missing.

Reply:
I do not know how to make the pipe change here. if you can add that in, that would be great! @mccalluc @ngehlenborg